homeassistant/components/binary_sensor/netatmo.py

```diff
@@ -44,14 +44,12 @@
 CONF_PRESENCE_SENSORS = 'presence_sensors'
 CONF_TAG_SENSORS = 'tag_sensors'
 
-DEFAULT_TIMEOUT = 15
-DEFAULT_OFFSET = 90
+DEFAULT_TIMEOUT = 90
```

Comment:
That is to high for a polling platform. Let's stick that to 10s.

Reply:
This is not a polling value, it is the time that the binary sensor will stay on after an event is detected. The polling of the platform is the default one (every 10s).